Jenkinsfile

```diff
@@ -340,7 +340,11 @@ pipeline {
                     def relLogsDstDir = logsRelPathBasedOnTriggerSource(env.JOB_NAME,
                         env.BUILD_NUMBER, env.ZUUL_UUID)
                     def dst = logsDirInFilesystem(env.LOG_ROOT_DIR, env.LOG_SERVER_FOLDER, relLogsDstDir)
-                    publishDirToLogServer("to_publish", auth, dst)
+                    if(env.TRIGGERED_FROM_ZUULV3){
```

Comment:
As discussed in IRL, we could use the other parameter, `TENTACLE_ARCHIVE_URL` to determine trigger source.

Reply:
Done